dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/LogTraceData.java

```diff
@@ -10,13 +10,25 @@
  *******************************************************************************/
 package com.ibm.ws.logging.data;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 import com.ibm.ws.logging.collector.LogFieldConstants;
+import com.ibm.ws.logging.utils.SequenceNumber;
 
 /**
  *
  */
 public class LogTraceData extends GenericData {
 
+    private final SequenceNumber sequenceNumber = new SequenceNumber();
```

Comment:
this shouldn't be needed - you're only using what should be a static method on SequenceNumber